src/server/hotspot_algo_qps_variance.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2017, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <algorithm>
+#include <gtest/gtest_prod.h>
+#include <math.h>
+
+#include "hotspot_partition_data.h"
+#include "table_hotspot_policy.h"
+#include <dsn/perf_counter/perf_counter.h>
+
+namespace pegasus {
+namespace server {
+// PauTa Criterion
+class hotspot_algo_qps_variance : public hotspot_policy
```

Comment:
There is no other class inherit from hotspot_policy, why not combine the two classes?

Reply:
Solved in https://github.com/apache/incubator-pegasus/pull/597